docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,176 @@
+= Universal Configuration Store
+
+== Overview
+The Universal Configuration Store is a tool that enables the various configurations for a project to be centrally defined and managed. 
+It then provides a standardized way of accessing them, allowing the environment specific configurations to be dynamically provided to 
+their respective resources within the project at runtime.
+
+=== Setup
+To setup the configuration store one must <<defining-configuration-properties,define the configurations>> through properties files, optionally creating <<environment-specific-overrides,environment 
+specific overrides>>. Then the configuration store is ready for <<deploying-the-configuration-store,deployment>> to a Kubernetes cluster, using the <<backend-storage-types,backend storage type>>
+of choice.
+
+==== Defining Configuration Properties
+The configuration store utilizes the opensource project https://github.com/TechnologyBrewery/krausening[Krausening,role=external,window=_blank] for managing its configurations, where each configuration is 
+represented by a key value pair property. The key values pairs are stored as string types using the standard `.properties` file 
+format.
+
+Here is an example properties file:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=baseTopic
+----
+This file defines two configuration properties: `connector` and `topic`.
+
+TIP: For sensitive configuration values, Krausening supports https://github.com/TechnologyBrewery/krausening/blob/dev/krausening/README.md#krausening-in-four-pints-leveraging-jasypt-for-encryptingdecrypting-properties[encrypting and decrypting properties,role=external,window=_blank]
+
+===== Environment Specific Overrides
+In addition to the properties file introduced above, one can optionally specify a secondary properties file that houses environment
+specific configurations. These configurations will then used to override their respective base configurations.
+
+Continuing with the example above, suppose one defines an additional properties file like the following:
+[source,properties]
+----
+topic=ciTopic
+auth-token=auth-secret
+----
+
+Then when these two properties files are reconciled within the store, it results in a configuration equivalent to the following:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=ciTopic
+auth-token=auth-secret
+----
+
+NOTE: The `topic` property is overwritten when the environment specific value `ciTopic`.
+
+==== Deploying the Configuration Store
+The https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md[configuration store helm chart,role=external,window=_blank] is used for easily deploying it to a Kubernetes cluster. As part of the deployment, one must
```

Comment:
```suggestion
The https://github.com/boozallen/aissemble/blob/{git-tree}/extensions/extensions-helm/aissemble-configuration-store-chart/README.md[Configuration Store Helm chart,role=external,window=_blank] can easily deploy the store to a Kubernetes cluster. As part of the deployment, one must
```

Reply:
Not sure if we're capitalizing "Configuration Store". I'm fine with either, but Helm should be capitalized.

Reply:
Capitalized all helms and config stores